manager/controllers/app/policy_compiler_interface.go

```diff
@@ -33,11 +34,11 @@ func ConstructApplicationContext(datasetID string, input *app.M4DApplication, op
 }
 
 // TempLookupPolicyDecision is a temporary function used to simulate the policy compiler, until the policy
-// compiler is updated to receive the externalDataset in place of the datasetID for external data sets.
+// compiler is updated to handle uncataloged data.
 // TODO: Update LookupPolicyDecisions after policy compiler is updated
-func TempLookupPolicyDecision(externalDataset app.ExternalDataContext, policyCompiler pc.IPolicyCompiler, req *modules.DataInfo, input *app.M4DApplication, op pb.AccessOperation_AccessType) error {
-	// For now approve if destGeo is the same as the residency of the data set
-	if req.Geo == string(externalDataset.Residency) {
+func TempLookupPolicyDecision(dataset app.DataContext, policyCompiler pc.IPolicyCompiler, req *modules.DataInfo, input *app.M4DApplication, op pb.AccessOperation_AccessType) error {
+	// For now approve if destGeo is Turkey
```

Comment:
I suggest to remove this hard-coded function, and compare instead req.Geo with whatever you got from the catalog connector in DatasetDetails (there is some Geo there as well)

Reply:
Will do so when policies are finalized